mikeio/pfs.py

```diff
@@ -453,7 +456,9 @@ def _parse_line(self, line: str) -> str:
                 key = key.strip()
                 value = s[(idx + 1) :]
 
-                if s.count("'") == 2:  # This is a quoted string and not a list
+                if (
+                    s[0] == "'" and s[-1] == "'"
```

Comment:
keyword = "str1", "str2"

    keyword = "str1", 0.0, false, "str2"

I guess you still need the `s.count("'") == 2` condition?

Reply:
Go ahead and add a failing test 😉

Reply:
🚀 science